docs/content/deployment.mdx

```diff
@@ -42,4 +42,3 @@ This section provides a series of hands-on guides to walk you through Dagster de
 | [Deploying Dagster to GCP](/deployment/guides/gcp)         | This guide explains how to install and deploy Dagster to GCP GCE.                        |
 | [Executing on Celery](/deployment/guides/celery)           | This guide explains how to execute Dagster on Celery.                                    |
 | [Executing on Dask](/deployment/guides/dask)               | This guide explains how to execute Dagster on Dask.                                      |
-| [Executing on Airflow](/deployment/guides/airflow)         | This guide explains how to execute Dagster on Airflow.                                   |
```

Comment:
is the plan to remove this guide completely? if yes, we should remove `docs/content/deployment/guides/airflow.mdx` too?